### PR TITLE
[master] Build EVM-DS from this repo on the builder node

### PR DESCRIPTION
**Pull request by valeryz**
## Description
Followup to the PR merging the repositories

## Backward Compatibility

- [x] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [ ] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

**Comment by valeryz** [comment]
This needs to be tested once Jenkins is alive again.

**Comment by valeryz** [comment]
This has been tested by letting Jenkins build it and deploying a testnet, and making sure that the `evm-ds` binary is there.

File: docker/Dockerfile [hunk 1]
```diff
@@ -19,23 +19,6 @@ FROM ${BASE} AS scilla
...
-apt-get install -y pkg-config && \
-apt-get install -y libssl-dev && \
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o install_script.sh && \
-sh install_script.sh -y 
-ENV PATH="/root/.cargo/bin:${PATH}"
-RUN git clone https://github.com/Zilliqa/evm-ds /evm-ds && \
-    cd /evm-ds && \
-    git checkout 70dfb1488e39b7c1e5fab6f040916c32ee714838 && \
-    cargo build --release --package evm-ds && \
-    cargo test --verbose --release --package evm-ds
```
Comment (its-saeed) [inline docker/Dockerfile:36]: did you remove `cargo test` intentionally?

**Review by its-saeed — APPROVED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (valeryz) [reply docker/Dockerfile:36]: Yes, it should run on CI eventually, but not essential for building an image.

**Pull request merged**